src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$backroundColor: #eee;
+$cellSize: 100px;
+$cellGap: 1px;
+$calendarPadding: 10px;
+$cellFonts: "Arial", "Helvetica", sans-serif;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cellSize + $calendarPadding * 2;
+  padding: 10px;
+  gap: 1px;
+
+  $days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
```

Comment:
Should move all variables to separate file

Reply:
When I create a separate file for variables, the page in the browser gives an error stating that such variables do not exist for the file where they are used

Reply:
https://prnt.sc/Ylt6ks1i0d3q

https://prnt.sc/m-FmtKjNNPME

When I connect variables to the main file, an error occurs on the browser page and these variables are not found

Reply:
@PaninDmytro use relative path for your imported files like so `@import "./variables";`